src/main/resources/db/changelog/001-create-p-order.yaml

```
@@ -0,0 +1,59 @@
+databaseChangeLog:
+  - changeSet:
+      id: 001-create-p-order
+      author: lhk
+      changes:
+        - createTable:
+            tableName: p_order
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false } }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)